### PR TITLE
Adding bigmler fusion subcommand

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

File: bigmler/options/main.py [hunk 1]
```diff
@@ -270,6 +270,20 @@ def get_main_options(defaults=None, constants=None):
...
             'help': "Fields added to the prediction file."},
 
+        # Probability: Includes the probability associated to the prediction
+        '--probability': {
+            'action': 'store_true',
+            'dest': 'probability',
+            'default': defaults.get('probability', False),
+            'help': ("Adding the probability to predictions.")},
+
+        # Probability: Includes the probability associated to the prediction
```
Comment (jaor) [inline bigmler/options/main.py]: i guess you mean No-probabilty: Excludes?

File: bigmler/options/fusion.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        # If a BigML JSON file containing a Fusion
+        # structure is provided,
+        # the script will use it.
+        '--fusion-file': {
+            'action': 'store',
+            'dest': 'fusion_file',
+            'default': defaults.get('fusion_file', None),
+            'help': "BigML Fusion JSON structure file."},
+
+        # The path to a file containing PCA attributes.
```
Comment (jaor) [inline bigmler/options/fusion.py]: PCA -> fusion

File: bigmler/delete/dispatcher.py [hunk 3]
```diff
@@ -59,6 +61,7 @@
 }
 
 GROUP_RESOURCES = ["project", "execution"]
+COMPOSED_RESOURCES = ["cluster", "ensemble", "fusion"]
```
Comment (jaor) [inline bigmler/delete/dispatcher.py]: at the api level, we also have "composites" (free lists of models)
not sure if you have to take them into account when using
resources_by_type? (even if not supported directly by bigmler)

**Comment by mmerce** [comment]
tests passed. You can merge whenever you see fit.

**Pull request merged**